packages/web-components/@sfx/products/stories/index.ts

```diff
@@ -1,24 +1,24 @@
 import { storiesOf } from '@storybook/html';
 import { withKnobs, text } from '@storybook/addon-knobs';
 import { ProductModel } from '@sfx/product';
-import { PRODUCTS_RESPONSE_EVENT } from '@sfx/products';
+import { SAYT_PRODUCTS_RESPONSE_EVENT } from '@sfx/products';
 import { getDisplayCode, getProducts, productsResultsEvent, hidePrompt } from '../../../../../.storybook/common';
 
 function getProductsComponent(productsArray: ProductModel[] = []) {
   if (productsArray.length > 0) {
     const products = text('Products', JSON.stringify(productsArray));
-    return `<sfx-products products="${products}"></sfx-products>`
+    return `<sfx-products-base products="${products}"></sfx-products-base>`
   } else {
-    return '<sfx-products></sfx-products>';
+    return '<sfx-products-base></sfx-products-base>';
   }
 }
 
-const productsNotesMarkdownIntro = ` # SF-X Products Component
+const productsNotesMarkdownIntro = ` # SF-X Products Base Component
 
 [Package README](https://github.com/groupby/sfx-view/tree/master/packages/web-components/%40sfx/products "SF-X Products README").
 
 \`\`\`html
-<sfx-products></sfx-products>
+<sfx-products-base></sfx-products-base>
```

Comment:
I believe the story has to be updated to remove the Custom Event (`sfx::sayt_products_response`). Additionally, we'll need stories for the SAYT and Search version of `Products`.

Reply:
Yup, that's pretty much the WIP for storybook that i commented on slack. I should have commented that here as well.